catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/UserRepository.java

```diff
@@ -147,21 +161,40 @@ private User validateUser(UUID userId) throws IOException {
     return validatedTeams;
   }
 
+  /* Add all the roles that user has been assigned, to User entity */
+  private List<EntityReference> getRoles(User user) throws IOException {
+    List<String> roleIds = dao.relationshipDAO().findTo(user.getId().toString(), HAS.ordinal(), Entity.ROLE);
+    List<EntityReference> roles = new ArrayList<>(roleIds.size());
+    for (String roleId : roleIds) {
+      roles.add(dao.roleDAO().findEntityReferenceById(UUID.fromString(roleId)));
+    }
+    return roles;
+  }
```

Comment:
This method has overlapping code with `validateRoles` method. Maybe the code can be refactored for reuse?